src/tools/rbd_mirror/ImageReplayer.cc

```diff
@@ -225,6 +225,19 @@ class ImageReplayerAdminSocketHook : public AdminSocketHook {
   Commands commands;
 };
 
+int replay_delay(const utime_t &event_time, int mirroring_delay) {
+    if (mirroring_delay <= 0) {
+      return 0;
+    }
+
+    utime_t now = ceph_clock_now(g_ceph_context);
+    if (event_time + mirroring_delay <= now) {
+      return 0;
+    }
+
+    return (event_time + mirroring_delay - now) + 1;
```

Comment:
It is to convert a double to the smallest following int (I could use ceil() but thought my version was good enough for this purposes).